targets/targets.json

```diff
@@ -8880,7 +8938,7 @@
             "WATCHDOG"
         ],
         "release_versions": ["5"],
-        "extra_labels": ["Cypress", "PSOC6", "MXCRYPTO"],
+        "extra_labels": ["Cypress", "PSOC6"],
```

Comment:
@ccli8 @0xc0170 Seems like some issue with rebasing. Why is this PR trying to update other targets? Please check.

Reply:
Sorry. I fixed them. It may result after several rebase operations.